src/app/globals.css

```
@@ -37,7 +37,7 @@
   --sidebar-accent-foreground: oklch(0.3729 0.0306 259.7328);
   --sidebar-border: oklch(0.8717 0.0093 258.3382);
   --sidebar-ring: oklch(0.5854 0.2041 277.1173);
-  --font-sans: Inter, sans-serif;
+  --font-sans: var(--font-sans), Inter, sans-serif;
```

Stylelint (17.13.0)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)